core/src/main/java/org/sonarsource/sonarlint/core/telemetry/payload/TelemetryRulesPayload.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * SonarLint Core - Implementation
+ * Copyright (C) 2016-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.telemetry.payload;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.Collection;
+
+public class TelemetryRulesPayload {
+
+  @SerializedName("explicitly_enabled_rules")
```

Comment:
I would remove the "_rules" suffix since we are already in the "rules" object. Also maybe double check with others in the team is the names are good (valid english).